source/manuals/programming-languages/nodejs/index.html.md.erb

```diff
@@ -82,24 +116,339 @@ Keeping tests in the same directory makes them easier to find and more obvious w
 |   └── product.njk
 ```
 
-#### Use a config directory
-Keep your configuration together in a config folder.
 
+See also:
+
+- [Node.js Project Structure Tutorial](https://blog.risingstack.com/node-hero-node-js-project-structure-tutorial/) on [Rising Stack](https://blog.risingstack.com/)
+
+## Language constructs
```

Comment:
I was convinced when we added the first version that language conventions are probably not something we should stipulate in the GDS way. 
1) It's too prescriptive as the language develops.
2) Its too divisive to agree on
3) The development community at large on the interweb can probably do it better

By stipulating here we lose the advantage of having a prescriptive linter like `standards` or `xo` and end up with meetings about tabs, spaces, semicolons, const, let, var etc.

An exception is choice of libraries, such as `we use express 4.x as our server framework`, `we use mocha, chai and sinon as our testing framework`

Reply:
That being said, there is pretty much nothing I disagree with here in terms of chosen style.

Reply:
I think there is a place for recommendations that sit between style (space/tabs, comments, wrapping, etc) and frameworks (mocha, express, etc). I called it language constructs and while I'm all for copying existing conventions like xo and airbnb, they don't cover those constructs. It's difficult to strike a balance as to what to recommend and it could indeed lead to many discussions, but there's virtue in saying use class for classes, or async/await for promises, etc.

Reply:
I’m ok with everything in the _Language constructs_ section, I think it’s a good outline of sensible approaches that are good for consistency

👍